### PR TITLE
ENGDOCS-761 team onboarding

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Added Team onboarding topic.
Renamed TOC entry.
Added repo consolidation to Business onboarding.

### Related issues (optional)

ENGDOCS-761

### CODE REVIEW

**Review by izzychu — APPROVED** [review]

File: docker-hub/onboard-team.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+toc_max: 2
+---
+
+The following section contains step-by-step instructions on how to get started onboarding your organization after you obtain a Docker Team subscription.
+
+## Prerequisites
+
+Before you start to on board your organization, ensure that you've completed the following:
+- You have a Docker Team subscription. [Buy now](https://www.docker.com/pricing/) if you haven't subscribed to Docker Team yet.
+-  You are familiar with Docker terminology. If you discover any unfamiliar terms, see the [glossary](/glossary/#docker) or [FAQs](../docker-hub/onboarding-faqs.md).
```
Comment (jerae-duffin) [inline docker-hub/onboard-team.md:15]:
```suggestion
- You are familiar with Docker terminology. If you discover any unfamiliar terms, see the [glossary](/glossary/#docker) or [FAQs](../docker-hub/onboarding-faqs.md).
```

File: docker-hub/onboard-team.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Step 1: Identify your Docker users and their Docker accounts
+
+To begin, you should identify which users you will need to add to your Docker Team organization. Identifying your users will help you efficiently allocate your subscription's seats and manage access.
+
+1. Identify the Docker users in your organization.
+   - If your organization uses device management software, like MDM or JAMF, you may use the device management software to help identify Docker users. See your device management software's documentation for details. You can identify Docker users by checking if Docker Desktop is installed at the following location on each user's machine:
+      - Mac: `/Applications/Docker.app`
+      - Windows: `C:\Program Files\Docker\Docker`
+   - If your organization does not use device management software, you may survey your users.
```
Comment (jerae-duffin) [inline docker-hub/onboard-team.md:26]:
```suggestion
   - If your organization doesn't use device management software, you may survey your users.
```

**Comment by jerae-duffin** [comment]
LGTM @craig-osterhout few minor suggestions

**Review by jerae-duffin — APPROVED** [review]
LGTM

**Pull request merged**